docs/openapi/components/schemas/common/AgParcelDelivery.yml

```diff
@@ -216,7 +216,71 @@ example: |-
       "phoneNumber": "555-895-1661",
       "faxNumber": "555-497-2527"
     },
-    "AgPackage": [],
+    "AgPackage": [
+      {
+        "type": [
+          "AgPackage"
+        ],
+        "packageName": "Avocados, Bulk",
+        "grade": "AA",
+        "responsibleParty": {
+          "type": "Entity",
+          "name": "Example Responsible Party Organization",
+          "email": "Chadrick_Gibson@example.com",
+          "phoneNumber": "+1-555-820-1520",
+          "entityType": "Organization"
+        },
+        "voicePickCode": "4642",
+        "date": "2021-03-14",
+        "labelImageUrl": "https://img.example.org/640/480/",
```

Comment:
why not `label: { image, hash }` ?

Reply:
Good point, though this seems out of scope for this PR